ee/localserver/certs.go

```diff
@@ -152,25 +150,18 @@ func dt4aKeys() (map[string]*ecdsa.PublicKey, error) {
 	}
 
 	for k, v := range certMap {
-		// Note that a successful parsing of any type of key does NOT necessarily guarantee a valid key.
-		jwkKey, err := jwk.ParseKey(v)
-		if err != nil {
-			return nil, fmt.Errorf("failed to parse key, kid %s: %w", k, err)
-		}
 
-		var pubKey ecdsa.PublicKey
-		if err := jwkKey.Raw(&pubKey); err != nil {
-			return nil, fmt.Errorf("failed to convert public key to public ecdsa, kid %s: %w", k, err)
+		var jwk jwk
```

Comment:
`var j jwk` to avoid shadowing? (I did put in the linter changes -- https://github.com/kolide/launcher/pull/2155 -- but fixing violations is a bit of a sprawling changeset so I was going to wait on pushing those.)